src/languages/bigquery/bigquery.formatter.ts

```diff
@@ -6,19 +6,19 @@ import { keywords } from './bigquery.keywords';
 import { functions } from './bigquery.functions';
 
 const reservedCommands = expandPhrases([
-  // DQL, https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax
-  'SELECT',
+  // Queries: https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax
```

Comment:
replace with DQL to follow pattern of DDL, DML, DQL, DCL

Reply:
I specifically opted to not use DQL. These three-letter-acronymes are easy to mix up, and I always have to pause and think what they mean.